### ISSUE TITLE
Icon of console application doesn't get used

### ISSUE THREAD

**Issue by gyorokpeter** [microsoft/terminal#14372]
### Windows Terminal version

1.15.2875.0

### Windows build number

10.0.22621.819

### Other Software

FAR Manager or any console app with an embedded icon

### Steps to reproduce

Launch a console application that has an embedded icon in its executable, such as FAR Manager. The Windows update that causes every console application to launch in Windows Terminal (instead of the old console app) must be installed.

### Expected Behavior

The embedded icon in the executable should be used as the tab icon inside Windows Terminal.
The embedded icon in the executable should be used as the taskbar icon.

### Actual Behavior

The tab icon remains the default generic icon (which looks like `C:\` on a white background).
The taskbar icon remains the default generic icon (which looks like `>_` on a black background).

This makes it difficult to distinguish between different console applications.

**Comment by DHowett** [comment]
Good catch on us not having an issue for this... probably just a bullet point in some other place we never formally tracked. Thanks for filing it, @gyorokpeter!

**Comment by zadjii-msft** [comment]
Oh wait, this might actually be tracked in #7552
